helm/scylla-manager/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v2
-appVersion: v1.0.0
 name: scylla-manager
-version: 1.0.0
 description: Scylla Manager automates database operations.
+version: "0.0.0" # overwritten during publishing
+appVersion: "0.0.0" # overwritten during publishing
```

Comment:
nit: does appVersion require any convention for format? `as it was previously `v1.0.0`, maybe `v0.0.0`

Reply:
appVersion doesn't need semantic, `version` does. Both versions are accepted, with `v` and w/o